syft/messaging/plan.py

```diff
@@ -3,61 +3,50 @@
 from typing import List
 from typing import Tuple
 from typing import Union
+from typing import Dict
 
 import torch
```

Comment:
Since when is this file torch-specific? Maybe I missed this during my refactors.  It shouldn't be outside of the frameworks/torch folder if that's the case -- but since Plans should be general in theory, we'll need to rethink this in the future

Reply:
Should not be torch specific, but indeed was still depending on torch a bit, I'm working to get it completely indenpenedent